src/app/client/src/app/modules/dashboard/services/report/report.service.ts

```diff
@@ -53,20 +56,19 @@ export class ReportService {
     });
     return forkJoin(...apiCalls).pipe(
       mergeMap(response => {
+
+        response = response.filter(function(item){ if(item ){ return item.loaded = true } });
         return this.getFileMetaData(dataSources).pipe(
           map(metadata => {
             return _.map(response, res => {
               res.lastModifiedOn = _.get(metadata[res.id], 'lastModified');
               return res;
             });
           })
+          ,catchError(err => {
```

Comment:
Remove this code block if you're not doing anything with the error and rethrowing it again.

Reply:
removed